CotEditor/Sources/ScriptManager.swift

```diff
@@ -254,6 +321,20 @@ final class ScriptManager: NSObject, NSFilePresenter {
     }
     
     
+    private func loadScriptInfo(at url: URL) {
```

Comment:
I quitted to pass `nil` as the direct parameter because `nil` is regareded as the current application in the user script and that fact causes strange behaviors.